cogs/games.py

```diff
@@ -40,15 +41,16 @@ async def roll(self, ctx: Context, min_limit: int = 1, max_limit: int = 10) -> N
         """Roll a random number."""
         if max_limit - min_limit > 2:
             number = random.randint(min_limit, max_limit)
-            embed = Embed(title="Random Roll", color=Color.blue(), description=f"The random number is: {number}",)
+            embed = Embed(title="Random Roll", color=Color.blurple(), description=f"The random number is: {number}")
             await ctx.send(embed=embed)
         else:
-            embed = Embed(title="Random Roll", color=Color.red(), description="Please specify numbers with difference of **at least 2**",)
+            embed = Embed(title="Random Roll", color=Color.red(), description="Please specify numbers with difference of **at least 2**")
             await ctx.send(embed=embed)
 
     @command(aliases=["8ball"])
     async def ball8(self, ctx: Context, *, question: str) -> None:
         """Play 8ball."""
+        answer = ""
```

Comment:
Is this really needed? `answer` should always get set by one of the conditions above